### ISSUE TITLE
coadd crash

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#1153]
using desispec/0.39.1:
```
desi_coadd_spectra \
  -i /global/cfs/cdirs/desi/spectro/redux/cascades/tiles/80617/20201222/spectra-4-80617-20201222.fits \
  -o $SCRATCH/coadd-4-80617-20201222.fits
...
Traceback (most recent call last):
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/bin/desi_coadd_spectra", line 4, in <module>
    __import__('pkg_resources').run_script('desispec==0.39.1', 'desi_coadd_spectra')
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/conda/lib/python3.8/site-packages/pkg_resources/__init__.py", line 667, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/conda/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1464, in run_script
    exec(code, namespace, namespace)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/lib/python3.8/site-packages/desispec-0.39.1-py3.8.egg/EGG-INFO/scripts/desi_coadd_spectra", line 16, in <module>
    coadd_spectra.main(args)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/lib/python3.8/site-packages/desispec-0.39.1-py3.8.egg/desispec/scripts/coadd_spectra.py", line 139, in main
    coadd(spectra,cosmics_nsig=args.nsig)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/lib/python3.8/site-packages/desispec-0.39.1-py3.8.egg/desispec/coaddition.py", line 153, in coadd
    ttflux[bad] = np.interp(spectra.wave[b][bad],spectra.wave[b][good],ttflux[good])
  File "<__array_function__ internals>", line 5, in interp
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/conda/lib/python3.8/site-packages/numpy/lib/function_base.py", line 1423, in interp
    return interp_func(x, xp, fp, left, right)
ValueError: array of sample points is empty
```

**Comment by sbailey** [comment]
Other nights with missing coadds to investigate, possibly the same problem.  Suspiciously, most of these are on sp9:

spectra-1-80642-20210106.fits
spectra-9-80709-20210216.fits
spectra-9-80690-20210216.fits
spectra-9-80712-20210216.fits
spectra-9-80685-20210216.fits
spectra-9-80738-20210217.fits
spectra-9-80732-20210217.fits
spectra-9-80733-20210217.fits
spectra-9-80731-20210217.fits
spectra-9-80705-20210218.fits
spectra-9-80706-20210218.fits
spectra-9-80733-20210218.fits
spectra-9-80730-20210218.fits
spectra-9-80735-20210218.fits
spectra-9-80710-20210220.fits
spectra-9-80657-20210220.fits
spectra-9-80666-20210220.fits
spectra-9-80658-20210220.fits
spectra-9-80712-20210220.fits
spectra-9-80632-20210222.fits
spectra-9-80782-20210223.fits

**Comment by julienguy** [comment]
I have checked that all spectra are successful with the fixes in PR  #1163.
